clientv3/config.go

```diff
@@ -33,6 +33,10 @@ type Config struct {
 	// DialTimeout is the timeout for failing to establish a connection.
 	DialTimeout time.Duration `json:"dial-timeout"`
 
+	// DialKeepAlive is the time after which client pings the server to see if transport is
+	// alive. It waits for this duration for a response before closing the connection.
+	DialKeepAlive time.Duration `json:"dial-keep-alive"`
```

Comment:
`dial-keepalive-probe` ?
We can add the timeout too though, if you think its useful.